### PR TITLE
feat: add C implementation for `math/base/special/asind`

### PR DESCRIPTION
**Pull request by Userf1995**
Resolves # .

## Description

> What is the purpose of this pull request?

This pull request:

-   add C implementation for math/base/special/asind

## Related Issues

> Does this pull request have any related issues?

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [X] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/asind/include/stdlib/math/base/special/asind.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+/**
+* Computes the arcsine (in degrees) of a double-precision floating-point number.
+*/
+double stdlib_base_asind ( const double x );
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif // !STDLIB_MATH_BASE_SPECIAL_ASIND_H
```
Comment (Pranavchiku) [inline lib/node_modules/@stdlib/math/base/special/asind/include/stdlib/math/base/special/asind.h]: Add a trailing new line

File: lib/node_modules/@stdlib/math/base/special/asind/src/main.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+* Computes the arcsine (in degrees) of a double-precision floating-point number.
+*
+* @param x - input value
+* @returns arcsine (in degrees)
+*
+* @example
+* var v = asind( 0.0 );
+* // returns 0.0
```
Comment (Pranavchiku) [inline lib/node_modules/@stdlib/math/base/special/asind/src/main.c]: please refer other packages, this is javascript one.

**Review by Pranavchiku — CHANGES_REQUESTED** [review]
Left a few comments above, once addressed this will be good to merge.

**Comment by Userf1995** [comment]
@Pranavchiku I have addressed all the issues you mentioned in your review. Please take a look at the changes

**Review by Planeshifter — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**